### ISSUE TITLE
"OC : failed to drop ACPI 54445353" error on boot

### ISSUE THREAD

**Issue by bademjan** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#208]
Hi Chris,

First of all thanks for your great work !  Thanks to your guide the hackintosh I've been very happy with a fully functioning hackintosh for more than a year. 

My hack has the same specs as yours except that I use an AMD Radeon RX580 GPU instead of the W5500 eGPU. I'm currently using OC 0.6.2 from your 3.xx release, with the iMac20.2 config, and it's been working like a charm without having to modify much.

I'm now trying to update to a newer OC version  because I would be more confortable having a fresh one before upgrading to Monterey. And here is my issue : it's impossible to boot, I get a blackscreen before picker. With debug on I get the following error : 

"OC : failed to drop ACPI 54445353 3464736D635F6878 1387 (1) - Not Found"

I tried several different releases (master branch, v12, V6, V5, V4) and  the different config.plist files proposed in the release, and the issue still persists. 

Any clue about what to do ? 

Thanks a lot !

**Comment by SchmockLord** [comment]
Disable the ACPI/Delete and use a kext-based USB port config.

Somehow your ACPI doesn't have this table/ at this position and so the delete of the original USB port config can't be executed. I guess because you are on a different bios version than the one I did this config on.

**Comment by gtrooper** [comment]
Oh ! I tried to update yesterday and got exactly the same error ! I am using the latest BIOS in Gigabyte Designare Z490 ! Could you please let me know which BIOS you used to create the SSDT for USB ? Thanks a lot !

**Comment by bademjan** [comment]
Indeed I have bios version F20 because I couldn't find version F20b on the Gigabyte website, maybe it's the source of the problem. Thanks for the instructions I will try asap !

**Comment by gtrooper** [comment]
> Indeed I have bios version F20 because I couldn't find version F20b on the Gigabyte website, maybe it's the source of the problem. Thanks for the instructions I will try asap !

@bademjan There is a little confusion with BIOS Updates for this board. Gigabyte, for some obscure reason uploaded an F21 BIOS update on 24/11/2021.  BUT the obscure thing is that the F21 is actually NEWER than the BIOS F21b which got uploaded on 05/10/2021. So ... For some reason the F21 is NEWER than F21b ... LoL !  Anyway ! After Updating to the NEWEST BIOS F21, The Kextless usb port mapping works amazingly well ! A huge thanks to @SchmockLord  !  BUT I would like to point out that in order to be able to BOOT I HAD TO replace the SSDT-Disable-CNVW.aml from a previous OC Release (e.g. 0.7.5) Because the CURRENT ONE uploaded in OC 0.7.8 Does NOT Boot. ( It produces ACPI Error) ! I hope this clears the confusion !

**Comment by SchmockLord** [comment]
@gtrooper Thanks for pointing that out. Funny thing is, that the previous version of the SSDT-Disable-CNVW.aml showed me ACPI errors when I opened it in MacIASL. Can you open both versions and screenshot me what the compile status? 

And can you try to find out the ACPI-path to the cnvw device from the DSDT.aml?

**Comment by gtrooper** [comment]
@SchmockLord 

I just dumped the original DSDT.aml using OpenCore Debug 0.7.8 (And Newest BIOS F21), so I suppose that you can extract the information you need from this DSDT. If there is anything else I can do, I will be happy to help :)

[https://mega.nz/file/htgTnKiS#wdgebs9oMABATG6Xq1_3D0mBKC6cnYiPUrgB2jd7krs](url)

**Comment by ashishpandey** [comment]
> > Indeed I have bios version F20 because I couldn't find version F20b on the Gigabyte website, maybe it's the source of the problem. Thanks for the instructions I will try asap !
> 
> @bademjan There is a little confusion with BIOS Updates for this board. Gigabyte, for some obscure reason uploaded an F21 BIOS update on 24/11/2021. BUT the obscure thing is that the F21 is actually NEWER than the BIOS F21b which got uploaded on 05/10/2021. So ... For some reason the F21 is NEWER than F21b ... LoL 

This is just gigabyte naming. F21b is the beta bios. F21 is final, which is why it replaces F21.
For that matter, all bios versions ending in a letter are pre-release versions. To make matters worse, gigabyte sometimes leave it at pre-release version forever (or a long time) without clearly qualifying it as such

**Comment by AlaKru** [comment]
I finally got a new monitor and decided to do the kextless update, it booted but had missing usb devices again. Went ahead and did the F21 bios update and now it will not boot, only goes straight to black screen. Any thoughts?

**Comment by gtrooper** [comment]
@SchmockLord Did you have any time to check what actually happens with the SSDT-Disable-CNVW.aml ? Did you check my newly created DSDT ? Thank you for your time !

**Comment by gtrooper** [comment]
@SchmockLord Very Nice Work SchmockLord ! I just tested your 0.7.9 EFI with the latest changes and now it runs perfectly ! I use your SSDT Based USB Mapping ! The boot with the USB Kext based config is noticeably slower... so for those that do not have issues with your SSDT Based USB Mapping it is an amazing solution ! Just a question: Did I provide you with what you asked in the previous post ? Was my newly generated DSDT from BIOS F21 helpful at all ? Did you manage to find the ACPI path ? 

Also: I noticed that in your VERY EARLY Builds (Release 3 or older) you had set CORRECTLY the Device ID of the iGPU to "923E0000" which is the proper value for UHD 630 for the Desktop iGPU.  The Device-ID you use in your currents build is WRONGLY set to "C59B0000", which is the Mobile Version of the UHD 630 iGPU ! While this seems to NOT affect anything critical, it actually does... I discovered that it causes Weird Errors AND black screens when using SIDECAR ! With the correct Device ID, Sidecar is unbelievably smooth and works amazingly well ! Could you please make this change in your next Release ? Thank you so much for your valuable help !

Also: As mentioned in the OPEN ISSUE #195 a few hours ago, OpenCore 0.7.9 seems to have a new option SetApfsTrimTimeout. If you set this value to 0 it disables Trim upon booting (But it seems that it enables it again as soon as it completes booting) and this seems that  it enables crazy fast booting regardless of the  SSD Vendor. I tested booting speed with a Seagate PCIe 4.0 NVME  AND with a ADATA XPG PCIe 4.0 NVME and in BOTH Drives it reduced booting from 20 seconds to 9 seconds ! Also this works in Catalina and NOT ONLY Monterey+ ! Do you see any disadvantage setting the default value to 0 instead of -1 ? Thank you so much AGAIN for EVERYTHING !

**Comment by SchmockLord** [comment]
@gtrooper Yes, the DSDT.aml helped, thanks. I just went back to an old SSDT-Diable-CNVW.aml where the PCI-Path was correct already.

Thanks for the hint with the iGPU device-ID. Just fixed that.

And I changed the SetApfsTrimTimeout to 0 in all the configs. 

It worked on my Z590i with a 2TB Intel 660p, blazing fast compared to before. Don't see any disadvantage. Seems to solve the issue with Trim we had before.

Can you try my newest commit?

**Comment by gtrooper** [comment]
I just did !  A-M-A-Z-I-N-G ! :) I think that we are ready for a Final Release ! (Although I did not install Monterey 12.3 yet to check what happens with the latest bugs with AMD 5000-6000 GPU's, but I think that you run 12.3 in your Z590i, right?)  Seems that Universal Control is too good of a feature to stay away from it for me :) I ll take the plug soon ! Thank you so much again !!!

**Comment by AlaKru** [comment]
Seems like you guys made a lot of progress. Can’t wait to try it. I have the Samsung 980 pro NVMe M.2 and the boot times were putting  me to sleep

**Comment by AlaKru** [comment]
> @SchmockLord I just did ! A-M-A-Z-I-N-G ! :) I think that we are ready for a Final Release ! (Although I did not install Monterey 12.3 yet to check what happens with the latest bugs with AMD 5000-6000 GPU's, but I think that you run 12.3 in your Z590i, right?) Seems that Universal Control is too good of a feature to stay away from it for me :) I ll take the plug soon ! Thank you so much again !!!

Looks like they may have made progress in #212 with the Gpu also.

**Comment by AlaKru** [comment]
@gtrooper Are you on Monterey? Any issues with i225 Ethernet?

**Comment by gtrooper** [comment]
@AlaKru I still use Catalina but I just installed Monterey 12.3 on an external SSD and the i225 Ethernet works like a charm. Do you still experiencing issues with it ?

**Comment by AlaKru** [comment]
@gtrooper Yes, I got a Samsung crg9 and did away with the apple thunderbolt display so I attempted to just use Schmocklord's setup with his mapping, and lost the i225v ethernet, and then tried Monterey which still did not work. Went back to Big Sur and restored from backup but lost several applications and still have not found a solution for the ethernet. I could not tag you in #210, Sorry for crossing issues. But everything you guy's did has sped the rest of my system up, so that's good.

**Comment by bademjan** [comment]
If it's any help to ppl reading this :
I figured out what the initial problem was. I noticed that my Seagate Barracuda HDD was causing the issue, since when unplugged the system could boot properly. So I tried to replace the Openhfsplus.efi driver with the HFSplus.efi from the earlier builds and IT WORKED ! Don't really know why OpenHFS couldn't handle the HDD but anyways... I'm glad I can now enjoy Monterey with OC 0.8.4 (V23) 😎
